SUPPORT_MATRIX.md

```diff
@@ -0,0 +1,24 @@
+# Support matrix
+
+NOTE: File auto-generated by TestCheckSupportMatrix in main_test.go. Please
+don't modify manually.
+
+Feature |py2 | py2-cffi | py3-cffi | pypy2-cffi | pypy3-cffi
+--- | --- | --- | --- | --- | ---
+_examples/arrays | no | yes | yes | yes | yes
+_examples/cgo | yes | yes | yes | yes | yes
+_examples/consts | yes | yes | yes | yes | yes
+_examples/empty | yes | yes | yes | yes | yes
+_examples/funcs | yes | no | no | no | no
+_examples/gostrings | yes | yes | yes | yes | yes
+_examples/hi | yes | no | no | no | no
```

Comment:
FYI, This is because of limited supporting by CFFI API.
To my knowledge, there is no way to access buffer interface through CFFI API.
But except that one, All the test are passed.
So I recommend to change it like this.
+_examples/hi | yes | yes(limited) | yes(limited) | yes(limited) | yes(limited)

Reply:
It is very difficult to figure what is supported / what is not supported. "limited" might not indicate what is supported. If we have concrete example of what works and what doesn't work, we can have another example of what is supported in cffi and run it as part of unit test. WDYT? If you can point me to what is supported with cffi, I can add it as part of another pull request.

Reply:
I am ok with just saying "yes/no".
`_examples/hi` started as a simple example (just printing "hi") and grew with every new feature I managed to implement.
then (after the first refactor of `gopy`) I had to split it up into multiple more manageable tests.
`_examples/hi` should go back to being a very simple test printing "hi" and possibly adding a few numbers (and one should extract the more sophisticated tests into other more focused ones)

Reply:
Does it make sense to do it as part of another pull request? I would like to restrict this to just documentation related.

Reply:
yes, splitting `_examples/hi` can be done in another PR :)